app/src/main/java/ch/epfl/sdp/blindly/profile/Profile3.kt

```diff
@@ -0,0 +1,47 @@
+package ch.epfl.sdp.blindly.profile
+
+import android.content.Intent
+import android.os.Build
+import android.os.Bundle
+import android.view.View
+import android.widget.DatePicker
+import android.widget.TextView
+import androidx.annotation.RequiresApi
+import androidx.appcompat.app.AppCompatActivity
+import ch.epfl.sdp.blindly.R
+import java.time.LocalDate
+import java.time.Period
+
+
+class Profile3 : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.set_profile_3)
+    }
+
+    @RequiresApi(Build.VERSION_CODES.O)
+    fun startProfile4(view: View) {
+        findViewById<TextView>(R.id.warning_p3).visibility = View.INVISIBLE
+        val datePicker: DatePicker = findViewById<View>(R.id.datePicker) as DatePicker
+        val day: Int = datePicker.dayOfMonth
+        val month: Int = datePicker.month + 1
+        val year: Int = datePicker.year
+        val age = getAge(year, month, day)
+        if (age < 18) {
```

Comment:
Magic number here, consider creating a constant value

Reply:
Corrected in the final commit.